scss/_patterns_card.scss

```diff
@@ -51,12 +42,59 @@
     }
   }
 
-  %p-card--highlighted {
-    @extend %p-card;
-    box-shadow: 0 1px 5px 1px transparentize($color-dark, .8);
-  }
-
   @include vf-p-card-default;
   @include vf-p-card-highlighted;
+  @include vf-p-card-muted-header;
   @include vf-p-card-overlay;
 }
+
+@mixin vf-p-card-default {
+  .p-card {
+    @extend %p-card;
+    border: 1px solid $color-mid-light;
+    padding: $spv-intra--scaleable - $px $spv-intra--scaleable $spv-intra--scaleable;
+  }
+}
+
+@mixin vf-p-card-highlighted {
+  .p-card--highlighted {
+    @extend %highlight;
+    @extend %p-card;
+  }
+}
+
+@mixin vf-p-card-muted-header {
```

Comment:
Can you think of a name to describe the purpose/effect of the use of this pattern. Muted-heading is describing the styling but in the future that may change but the role of the pattern should not. Maybe even replace the highlight card?

Reply:
Just had a chat with @Caleb-Ellis and this led to a larger topic. I think we should discuss in the WG meeting, so I'll add a don't merge label on this pr. Caleb already merged a simpler fix that deals with the reported bug on those cards.

In short - the elements that are currently found in cards are miscellaneous. Their styling is also not specific to a card, and could be used in other layouts. The current card examples, together with the resources and insights cards, are made up of:

• muted heading followed by a rule (current resources card)
• 2rem thumbnail image followed by a rule
• h3 / h4 with anchor
• image
• p-list 
• subscribe form with p-list inside
• button
• any combination of the above

I think it would be simpler to limit the card to just a wrapper with padding, which has modifiers for border and box-shadow. And then compose cards as necessary. 

To avoid breaking things, the current .p-card__* classes could just point to atomic placeholders / classes that format them so they look as they do now.